### ISSUE TITLE
Greska evaluacije

### ISSUE THREAD

**Issue by DenisAlicic** [djanluka/FuzzyClassification#2]
Rezultati nisu validni iz razloga sto prvo normalizujete sve podatke, pa zatim radite train_test_split().

Tu ste vec izvukli neku informaciju iz train skupa, time sto ste "naucili" kako da normalizujete.

Prvo train_test_split, pa zatim "fit" nad train, zatim "transform" nad oba.

**Comment by djanluka** [comment]
Ispravio sam normalizaciju koju koristim u KNN-u i Naive Bayes-u. Ukoliko je u redu, mozemo da zatvorimo issue.
